src/__tests__/Company.test.js

```diff
@@ -0,0 +1,19 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import Company from '../components/Company';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <Router>
+          <Company />
+        </Router>
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Please, add more unit tests for several components of the app and some pure functions in Redux code.
- [ ] Some of your tests are failing. kindly work on it.
![Screenshot from 2022-03-11 00-40-58](https://user-images.githubusercontent.com/65860678/157767484-331d4ffd-7737-4d85-907e-feeaf7229732.png)

Reply:
Thank you for your review :+1: I have implemented the required changes :+1: